docs/content/en/getting-started/installing.md

```diff
@@ -205,7 +205,7 @@ Replace `brew install hugo` with `brew install hugo --HEAD` if you want the abso
 `brew` should have updated your path to include Hugo. You can confirm by opening a new terminal window and running a few commands:
 
 ```
-$ # show the location of the hugo executable
+# show the location of the hugo executable
```

Comment:
`$` is often used to emulate the shell console, but there are no other examples of such usage in the documentation, so it seems more consistent to remove this one